File: de.upb.crypto.zeroknowledge.parent/de.upb.crypto.zeroknowledge.web/WebRoot/console.js
/*
Functions for displaying messages in the console box
*/
function getConsole() {
    return document.getElementById("console");
}

// Writes an error message to the console box in the lower right corner
function consoleError(message) {
    getConsole().innerHTML = "<span class='error'>Error: </span>" + message;
}

// Writes a warning message to the console box in the lower right corner
function consoleWarning(message) {
    getConsole().innerHTML = "<span class='warning'>Warning: </span>" + message;
}

// Writes an info message to the console box in the lower right corner
function consoleInfo(message) {
    getConsole().innerHTML = "<span class='info'>Info: </span>" + message;
}

File: org.cryptimeleon.zeroknowledge.parent/org.cryptimeleon.zeroknowledge/src/org/cryptimeleon/zeroknowledge/console/Console.java
package org.cryptimeleon.zeroknowledge.console;

/**
 * A class for sending messages to the compiler website's console box.
 * Currently unused and not working.
 */
public class Console {

	public static void warning(String message) {
		System.out.println("Warning: " + message);
	}
	
	
	public static void error(String message) {
		System.out.println("Error: " + message);
	}
	
	public static void info(String message) {
		System.out.println("Info: " + message);
	}
}
